docs/docs/developers/reference/environment_reference/cli_reference.md

```diff
@@ -4,34 +4,208 @@ tags: [sandbox]
 sidebar_position: 2
 ---
 
-:::warning
+This reference guide provides documentation for the Aztec CLI commands (`aztec`) and their options. The CLI is a powerful tool for interacting with the Aztec network, managing accounts, deploying contracts, and more.
 
-`aztec-builder` and `aztec-sandbox` have been deprecated in favor of `aztec` CLI
+Consider using the [`aztec-wallet`](./cli_wallet_reference.md) for account related or interacting with an existing network (e.g. testnet).
 
-:::
-- [Start](#starting-and-testing)
-- [Accounts](#account-management)
-- [Contract deployments and interaction](#contract-deployment-and-interaction)
-- [Network and node info](#network-and-node-information)
-- [Querying](#transaction-and-block-querying)
-- [Logging](#logging-and-data-retrieval)
-- [Debugging](#development-and-debugging-tools)
-- [L1 contracts](#l1-contract-management)
-- [Utils](#utility-commands)
+## Overview
 
-## Starting and testing
+The Aztec CLI provides commands for:
+
+- **Starting and Testing**: Starting the Aztec Sandbox and running tests
+- **Account Management**: Creating, deploying, and managing Aztec accounts
+- **Contract Operations**: Deploying, interacting with, and managing smart contracts
+- **Network Information**: Querying node and network status
+- **Transaction Management**: Sending, canceling, and querying transactions
+- **Data Retrieval**: Accessing logs and contract data
+- **Development Tools**: Profiling, debugging, and code generation
+- **L1 Integration**: Managing L1 contracts and bridges
+- **Governance**: Participating in protocol governance
+- **P2P Network**: Managing peer-to-peer network configuration
+- **Utilities**: Various helper commands for development
+
+Each command section includes detailed options and examples of usage. The documentation is organized to help you quickly find the commands you need for your specific use case.
+
+Note: Most commands accept a `--rpc-url` option to specify the Aztec node URL, and many accept fee-related options for gas limit and price configuration.
+
+## Common Commands
+
+- [`aztec get-node-info`](#get-node-info)
+- [`aztec get-l1-addresses`](#get-l1-addresses)
+- [`aztec get-block`](#get-block)
+- [`aztec start --pxe`](#pxe-options)
+
+## Starting
 
 ### start
 
 Initiates various Aztec modules. It can be used to start individual components or the entire Aztec Sandbox.
 
-```
+```bash
 aztec start [options]
 ```
 
 Options:
+
+#### Network Options
+
+- `--network <value>`: Network to run Aztec on.
+
+#### Sandbox Options
```

Comment:
can we move this section first? (before network)?

Reply:
yep